Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml

```diff
@@ -45,7 +45,19 @@
   cost: 1500
   category: Livestock
   group: market
-  
+
+- type: cargoProduct
+  name: "duck crate"
+  id: LivestockDuck
+  description: "A crate containing a six fully grown ducks"
```

Comment:
```suggestion
  description: "A crate containing six fully grown ducks"
```

Reply:
Ah, that was an issue with the chicken description then as well, since that's what I copied